File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/LoginUserResponse.java
package com.rolandvitezhu.todocloud.network.api.user.dto;

import com.google.gson.annotations.SerializedName;
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse;

public class LoginUserResponse extends BaseResponse {

  @SerializedName("user_online_id")
  public String userOnlineId;
  @SerializedName("name")
  public String name;
  @SerializedName("email")
  public String email;
  @SerializedName("api_key")
  public String apiKey;

  public String getUserOnlineId() {
    return userOnlineId;
  }

  public void setUserOnlineId(String userOnlineId) {
    this.userOnlineId = userOnlineId;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }

  public String getApiKey() {
    return apiKey;
  }

  public void setApiKey(String apiKey) {
    this.apiKey = apiKey;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/LoginUserResponse.kt
package com.rolandvitezhu.todocloud.network.api.user.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LoginUserResponse (
        @SerializedName("user_online_id")
        var userOnlineId: String?,
        @SerializedName("name")
        var name: String?,
        @SerializedName("email")
        var email: String?,
        @SerializedName("api_key")
        var apiKey: String?
) : Parcelable, BaseResponse() {
    constructor() : this(
            null,
            null,
            null,
            null
    )
}